edx_rest_framework_extensions/auth/jwt/tests/test_authentication.py

```diff
@@ -191,6 +196,20 @@ def test_authenticate_csrf_protected(self, mock_set_custom_attribute):
             "Exception:PermissionDenied('CSRF Failed: CSRF cookie not set.')",
         )
 
+    def test_authenticate_with_disabled_user(self):
+        """"""
+        jwt_token = self._get_test_jwt_token()
+        request = RequestFactory().get('/', HTTP_AUTHORIZATION=jwt_token)
+
+        username = 'ckramer'
+        email = 'ckramer@hotmail.com'
+        user = factories.UserFactory(email=email, username=username, is_staff=False)
+
+        with mock.patch.object(JSONWebTokenAuthentication, 'authenticate', return_value=(user, "mock-auth")):
+            with mock.patch.object(User, 'has_usable_password', return_value=False):
+                with self.assertRaises(AuthenticationFailed):
+                    JwtAuthentication().authenticate(request)
```

Comment:
There are lot of AuthenticationFailed error raising from this class, can we please check the error code?

Reply:
Done